<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>772</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>772</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Poker Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tbBackground">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>26</y>
      <width>581</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/2.jpg);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCall">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
border-image: url(:/images/images/PokerTableBackground.jpg);
color: rgb(239, 239, 239);</string>
    </property>
    <property name="text">
     <string>Call</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFold">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>452</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
color: rgb(255, 255, 255);
border-image: url(:/images/images/PokerTableBackground.jpg);
</string>
    </property>
    <property name="text">
     <string>Fold</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRaise">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Raise Amount&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Copperplate Gothic Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCheck">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>424</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
color: rgb(255, 255, 255);
border-image: url(:/images/images/PokerTableBackground.jpg);</string>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNewGame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:400;&quot;&gt;Start a new Game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbQuitGame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Quit game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePlayerBalance">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>507</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Your Balance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
color: rgb(255, 255, 255);
background-image: url(:/images/images/PokerBackground.jpg);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePlayerName">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>437</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 10pt &quot;Copperplate Gothic Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAIName">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>58</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 10pt &quot;Copperplate Gothic Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAIBalance">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>127</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Your Balance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
background-color: rgb(0, 85, 0);
background-image: url(:/images/images/PokerBackground.jpg);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvPlayerCard1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>427</y>
      <width>73</width>
      <height>99</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvPlayerCard2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>427</y>
      <width>73</width>
      <height>99</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvAICard1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>47</y>
      <width>73</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvAICard2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>47</y>
      <width>73</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvTableCard2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>237</y>
      <width>73</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvTableCard1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>237</y>
      <width>73</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvTableCard4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>237</y>
      <width>73</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvTableCard3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>237</y>
      <width>73</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvTableCard5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>237</y>
      <width>73</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/b2fv.gif);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRaise">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>507</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
color: rgb(244, 244, 244);
border-image: url(:/images/images/PokerTableBackground.jpg);
</string>
    </property>
    <property name="text">
     <string>Raise</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvPlayerImage">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>437</y>
      <width>46</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/Character Face.jpg);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvAIImage">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>57</y>
      <width>46</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/Character Face.jpg);
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePotBalance">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>341</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Copperplate Gothic Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvPlayerPokerChips">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/PokerChips.jpg);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvAIPokerChips">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/PokerChips.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAIBetValue">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>482</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(172, 172, 172);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="teAIMoves">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>26</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Copperplate Gothic Bold&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="tePlayerMoves">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>355</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Copperplate Gothic Bold&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbNewGame</tabstop>
  <tabstop>pbQuitGame</tabstop>
  <tabstop>pbCheck</tabstop>
  <tabstop>pbCall</tabstop>
  <tabstop>pbFold</tabstop>
  <tabstop>leRaise</tabstop>
  <tabstop>pbRaise</tabstop>
  <tabstop>lePlayerBalance</tabstop>
  <tabstop>lePlayerName</tabstop>
  <tabstop>leAIName</tabstop>
  <tabstop>leAIBalance</tabstop>
  <tabstop>gvPlayerCard1</tabstop>
  <tabstop>gvPlayerCard2</tabstop>
  <tabstop>gvAICard1</tabstop>
  <tabstop>gvAICard2</tabstop>
  <tabstop>gvTableCard2</tabstop>
  <tabstop>gvTableCard1</tabstop>
  <tabstop>gvTableCard4</tabstop>
  <tabstop>gvTableCard3</tabstop>
  <tabstop>gvTableCard5</tabstop>
  <tabstop>tbBackground</tabstop>
  <tabstop>gvPlayerImage</tabstop>
  <tabstop>gvAIImage</tabstop>
  <tabstop>lePotBalance</tabstop>
  <tabstop>gvPlayerPokerChips</tabstop>
  <tabstop>gvAIPokerChips</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
